linux_os/guide/system/software/integrity/crypto/configure_openssl_crypto_policy/ansible/shared.yml

```diff
@@ -3,34 +3,50 @@
 # strategy = unknown
 # complexity = low
 # disruption = medium
+
+{{% set openssl_cnf_file="openssl.cnf" %}}
 {{% if 'sle' in product %}}
-  {{% set openssl_cnf_path="/etc/ssl/openssl.cnf" %}}
+  {{% set openssl_cnf_dir="/etc/ssl" %}}
+{{% else %}}
+  {{%- set openssl_cnf_dir="/etc/pki/tls" %}}
+{{% endif %}}
+
+{{% if product in ["fedora", "ol9", "rhel9"] %}}
+    {{% set ansible_openssl_include_directive = ".include = /etc/crypto-policies/back-ends/opensslcnf.config" %}}
 {{% else %}}
-  {{%- set openssl_cnf_path="/etc/pki/tls/openssl.cnf" %}}
+    {{% set ansible_openssl_include_directive = ".include /etc/crypto-policies/back-ends/opensslcnf.config" %}}
 {{% endif %}}
+{{% set full_config_section = "[crypto_policy]\n" + ansible_openssl_include_directive %}}
+
 
-- name: "Test for crypto_policy group"
-  command: grep '^\s*\[\s*crypto_policy\s*]' {{{ openssl_cnf_path }}}
+- name: "{{{ rule_title }}} - Test for crypto_policy group"
```

Comment:
```suggestion
- name: "{{{ rule_title }}} - Search for crypto_policy Section"
```

Reply:
Tasks names should be in Title Case: https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible

Reply:
I have accepted your suggestions.